library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -711,6 +711,14 @@ public void emitTuples()
   }
 
   /**
+   * Whether or not to advance the window now regardless of emitBatchSize.
+   */
+  protected boolean advanceWindow()
```

Comment:
The method does not "advance the window" and "window" by itself is ambiguous. Would "suspendEmit()" or something similar better convey the purpose? Maybe also expand on that in the javadoc?